be/src/exec/pipeline/exchange/sink_buffer.h

```diff
@@ -80,9 +129,17 @@ class SinkBuffer {
         }
     }
 
-    bool is_full() const { return _pending_chunks.get_size() >= config::pipeline_io_cache_size; }
+    bool is_full() const {
+        // TODO(hcf) if one channel is congested, it may cause all other channel unwritable
+        for (auto* cache : _caches) {
+            if (cache->size() > config::pipeline_io_cache_size) {
```

Comment:
Is this needn't protected by _mutex?

Reply:
It's ok that it may be not that accurate and it is harmless